docs/howto/catalog_exports.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Data Catalogs Export
+description: This section explains how lakeFS can integrate with external Data Catalogs via metastore update operations. 
+parent: How-To
+---
+
+# Data Catalogs Export
+
+{% include toc_2-3.html %}
+
+## About Data Catalogs Export
+
+Data Catalog Export is all about integrating query engines (like Spark, AWS Athena, Presto, etc.) with lakeFS.
+
+Data Catalogs (such as Hive Metastore or AWS Glue) store metadata for services (such as Spark, Trino and Athena). They contain metadata such as the location of the table, information about columns, partitions and much more.
+
+With Data Catalog Exports, one can leverage the versioning capabilities of lakeFS in external data warehouses and query engines to access tables with branches and commits. 
+
+At the end of this guide, you will be able to query lakeFS data from Athena, Trino and other catalog-dependent tools:
+
+```sql
+use main;
+use my_branch; -- any branch
+use v101; -- or tag
+
+SELECT * FROM users 
+INNER JOIN events 
+ON users.id = events.user_id; -- SQL stays the same, branch or tag exist as schema
+```
+
+## How it works 
+
+There are several well known formats that exist today that could help export existing tables in lakeFS into a "native" object store representation
+which don't require copying the data outside of lakeFS.
+
+Since these are metadata representations, they are applied using hooks to automate the process.
+
+### Table Decleration 
+
+Once a a lakeFS repository is created, the tables should be configured as a table descriptor object on the repository on the path `_lakefs_tables/TABLE.yaml`.
+Note: only tables of `type: hive` are supported and more are expected to be added. 
+
+#### Hive tables
+
+Hive metadata server tables are essentially just a set of objects that share a prefix, with no table metadata stored on the object store.  You need to configure prefix, partitions, and schema.
+
+```yaml
+name: animals
+type: hive
+path: path/to/animals/
+partition_columns: ['year']
+schema:
+  type: struct
+  fields:
+    - name: year
+      type: integer
+      nullable: false
+      metadata: {}
+    - name: page
+      type: string
+      nullable: false
+      metadata: {}
+    - name: site
+      type: string
+      nullable: true
+      metadata:
+        comment: a comment about this column
+```
+
+Note: Useful types recognized by Hive include `integer`, `long`, `short`, `string`, `double`, `float`, `date`, and `timestamp`.
+{: .note }
+
+### Catalog Exporters 
+
+Exporters are code packages written in lua, each exporter is exposed as a lua function under the package namespace `lakefs/catalogexport`, they are reusable hooks to connect various types of tables to different catalogs.
+[configuration]({% link reference/configuration.md %}).
+
+Note: Check the [lua Library reference]({% link howto/hooks/lua.md %}#lua-library-reference) library for code reference, specifically everything under the prefix `lakefs/catalogexport`. 
```

Comment:
I don't think we say "note" before a `{: .note }`.  I looked at 3 examples and none of them did that.

Reply:
👍